source/content/guides/php/01-introduction.md

```diff
@@ -32,17 +32,15 @@ Click the links below to display complete PHP information for each version, incl
 | [7.4](https://v74-php-info.pantheonsite.io/)     | <span style="color:green">✔</span>         | ❌          |
 | [7.3](https://v73-php-info.pantheonsite.io/)     | <span style="color:green">✔</span>         | ❌           |
 | [7.2](https://v72-php-info.pantheonsite.io/)     | <span style="color:green">✔</span>         | ❌           |
-| [7.1](https://v71-php-info.pantheonsite.io/)     | <span style="color:green">✔</span>          | ❌           |
-| [7.0](https://v70-php-info.pantheonsite.io/)     | <span style="color:green">✔</span>         | ❌           |
-| [5.6](https://v56-php-info.pantheonsite.io/)     | <span style="color:green">✔</span>         | ❌           |
+| [7.1](https://v71-php-info.pantheonsite.io/) <sup>2</sup>   | <span style="color:green">✔</span>          | ❌           |
+| [7.0](https://v70-php-info.pantheonsite.io/) <sup>2</sup>   | <span style="color:green">✔</span>         | ❌           |
+| [5.6](https://v56-php-info.pantheonsite.io/) <sup>2</sup>   | <span style="color:green">✔</span>         | ❌           |
 
-Sites that run older PHP versions not listed above will continue to serve pages. However, new development cannot be done because the development environment behavior is undefined and no longer supported. You can [upgrade your PHP version](/guides/php/php-versions) in the development environment to resume development on your site.
+Sites that run older PHP versions not listed above will continue to serve pages. While older sites will still run unlisted and unsupported versions of PHP, new sites cannot change their PHP version to unsupported values. You can [upgrade your PHP version](/guides/php/php-versions) in the development environment to resume development on your site.
 
-<Alert title="PHP 8.3 New Relic compatibility" type="info">
+<p><sup>1</sup> Currently, New Relic does not support PHP 8.3. As such, you will not be able to view your New Relic dashboard on any site that has been updated to PHP 8.3. We will be updating our platform to support New Relic on PHP 8.3 sites as soon as a compatible New Relic release is available to us.</p>
```

Comment:
Currently, PHP 8.3 on Pantheon does not support New Relic. As such, you will not be able to view your New Relic dashboard on any site that has been updated to PHP 8.3. We will be updating our platform to support New Relic on PHP 8.3 sites soon.

Reply:
```suggestion
<p><sup>1</sup> Currently, PHP 8.3 on Pantheon does not support New Relic. As such, you will not be able to view your New Relic dashboard on any site that has been updated to PHP 8.3. We will be updating our platform to support New Relic on PHP 8.3 sites soon.</p>
```